static/src/js/containers/DownloadHistoryContainer/__tests__/DownloadHistoryContainer.test.js

```diff
@@ -1,89 +1,250 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-
-import actions from '../../../actions'
 import {
-  DownloadHistoryContainer,
-  mapDispatchToProps,
-  mapStateToProps
-} from '../DownloadHistoryContainer'
+  render,
+  waitFor,
+  act
+} from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import { Provider } from 'react-redux'
+import configureStore from 'redux-mock-store'
+import nock from 'nock'
+
+import { addToast } from '../../../util/addToast'
+import { handleError } from '../../../actions/errors'
+import DownloadHistoryContainer from '../DownloadHistoryContainer'
 import { DownloadHistory } from '../../../components/DownloadHistory/DownloadHistory'
 
-Enzyme.configure({ adapter: new Adapter() })
+jest.mock('../../../util/addToast', () => ({
+  addToast: jest.fn()
+}))
+
+jest.mock('../../../actions/errors', () => ({
+  handleError: jest.fn(() => ({ type: 'HANDLE_ERROR' }))
+}))
+
+let lastPassedOnDeleteRetrieval = null
+
+jest.mock('../../../components/DownloadHistory/DownloadHistory', () => ({
+  DownloadHistory: jest.fn((props) => {
+    lastPassedOnDeleteRetrieval = props.onDeleteRetrieval
+
+    return <div>Download History Component</div>
+  })
+}))
 
-function setup(props) {
-  const enzymeWrapper = shallow(<DownloadHistoryContainer {...props} />)
+const mockStore = configureStore([])
 
-  return {
-    enzymeWrapper,
-    props
-  }
-}
+describe('DownloadHistoryContainer component', () => {
+  let store
```

Comment:
This `store` should be created in a setup function

Reply:
I don't think we need this store at all, if you look at our other container tests we import the named export of the component, not the default export. You are trying to test redux here functionality here and we only want to test our component code.